### PR TITLE
Tiny changes to signature stuff

### PR DESCRIPTION
**Pull request by jozefg**
Normally I wouldn't PR this but I want to make sure I'm saying true things in the documentation :) I also fixed two small bugs

### CODE REVIEW

File: src/signature/abt_signature.sml [hunk 1]
```diff
@@ -55,6 +55,9 @@ struct
       go Y1
     end
 
+  (* def is *almost* an identity, but it also does all the checking
+   * necessary to make sure that everything is well-sorted before hand
```
Comment (jonsterling) [inline src/signature/abt_signature.sml]: well-sorted and well-scoped

File: src/signature/morphism.sig [hunk 2]
```diff
@@ -1,8 +1,20 @@
+(* A signature morphism is the core part of a translation between two
+ * phases of a signature. Each portion of elaboration boils down to a
+ * different implementation of SIGNATURE_MORPHISM
```
Comment (jonsterling) [inline src/signature/morphism.sig]: perfect

File: src/signature/morphism.sig [hunk 3]
```diff
@@ -1,8 +1,20 @@
...
+ *)
 signature SIGNATURE_MORPHISM =
 sig
+  (* Source *)
   structure S1 : SIGNATURE
+  (* Target *)
   structure S2 : SIGNATURE
 
+  (* In order to implement this transformation this is the
+   * only required function to implement. It describes how to
```
Comment (jonsterling) [inline src/signature/morphism.sig]: Maybe rephrase as:

> In order to implement this transformation, it suffices to implement this single function.

File: src/signature/morphism.sig [hunk 4]
```diff
@@ -1,8 +1,20 @@
...
+   * elaborate one sorce declaration given all the rest of the
```
Comment (jonsterling) [inline src/signature/morphism.sig]: `sorce` => `source`

File: src/signature/morphism.sig [hunk 5]
```diff
@@ -1,8 +1,20 @@
...
+   * previously elaborated declarations. See TransportSignature
+   * for how this is elaborated into a map between two full
```
Comment (jonsterling) [inline src/signature/morphism.sig]: let's say `extended` instead of `elaborated` in this sentence

**Comment by jonsterling** [comment]
excellent, thank you very much for this! I have made a few little suggestions; other than that, it's good to go!

**Comment by jozefg** [comment]
Okay cool, I've fixed them!

**Comment by jonsterling** [comment]
great, thanks!!
